src/perfrecord.cpp

```diff
@@ -63,9 +69,14 @@ static QStringList sudoOptions(const QString& sudoBinary)
 {
     QStringList options;
     if (sudoBinary.endsWith(QLatin1String("/kdesudo")) || sudoBinary.endsWith(QLatin1String("/kdesu"))) {
+#if KWINDOWSYSTEM_VERSION >= QT_VERSION_CHECK(5, 101, 0)
+        auto activeWindowId = QString::number(KX11Extras::activeWindow());
```

Comment:
remove the QString::number from the two branches, only do that below in the `options.append()` line